### ISSUE TITLE
oo-few-public-methods reported on new @dataclasses.dataclass

### ISSUE THREAD

**Issue by dan-hook** [PyCQA/pylint#3618]
Looks like this bug still exists if the module name is included in the annotation.

### Steps to reproduce
1.

```python
import dataclasses

@dataclasses.dataclass
class ThisIsaDataclass():
    a_number: int
```

### Current behavior
too-few-public-methods reported

### Expected behavior
too-few-public-methods should not be reported

### pylint --version output
pylint 2.3.1
astroid 2.2.5
Python 3.7.4 (default, Aug 13 2019, 20:35:49) 
[GCC 7.3.0]

**Comment by PCManticore** [comment]
This has been fixed either in pylint 2.4.4 or in pylint 2.5.0. Please give it a go and let us know if you can still reproduce it.
